app/views/research_outputs/metadata_standards/_search.html.erb

```diff
@@ -7,8 +7,8 @@
 <%= render partial: "layouts/modal_search/form",
            locals: {
              namespace: "metadata_standards",
-             label: "Metadata Standard",
-             search_examples: "(e.g. DataCite, Dublin, Biological, etc.)",
+             label: _("Metadata Standard"),
+             search_examples: _("(e.g. DataCite, Dublin, Biological, etc.)"),
```

Comment:
Good catch! We should be taking note of these errors to see if we can share them through a PR on the latest version of roadmap.